pkg/kudoctl/util/repo/repo.go

```diff
@@ -57,6 +59,20 @@ func (r Repositories) CurrentRepo() *RepositoryConfiguration {
 	return r.GetRepo(r.Context)
 }
 
+// RepositoryConfig gets the repo configuration from settings
+func RepositoryConfig(fs afero.Fs, settings *env.Settings) (*RepositoryConfiguration, error) {
```

Comment:
could we not pass around whole generic settings? I would prefer to pass around only values that the method depends on, so in this case it's path to repo file and name of the repository. It also makes testing easier because you know what the method is using...